kuma/static/styles/components/content.styl

```diff
@@ -7,6 +7,13 @@
 
 $table-blue = #d4dde4;
 
+/*
+    Used to draw borders around objects which should only be bordered in
+    the editor.
+*/
+
+$editor-box-border-style = solid 1px #6699FF40;
```

Comment:
I could, however, make .hidden use its own variable in vars.styl so that it could theoretically be reused for other stuff (and to group these special bits of CSS in one place).